PWGHF/TableProducer/candidateSelectorOmegac0ToOmegaPi.cxx

```diff
@@ -124,6 +125,27 @@ struct HfCandidateSelectorToOmegaPi {
   Configurable<int> nClustersItsInnBarrMin{"nClustersItsInnBarrMin", 1, "Minimum number of ITS clusters in inner barrel requirement for pi <- charm baryon"};
   Configurable<float> itsChi2PerClusterMax{"itsChi2PerClusterMax", 36, "Maximum value of chi2 fit over ITS clusters for pi <- charm baryon"};
 
+  // KF selection
+  Configurable<bool> applyKFpreselections{"applyKFpreselections", true, "Apply KFParticle related rejection"};
+  Configurable<bool> applyCompetingCascRejection{"applyCompetingCascRejection", true, "Apply competing Xi(for Omegac0) rejection"};
```

Comment:
what about putting it to `false` by default? You mentioned that you'd consider it for PbPb to reduce the output size, but maybe in pp this is not necessary (of course: one can switch it on in analysis if desired)